### ISSUE TITLE
Can't locate YAML.pm in @INC

### ISSUE THREAD

**Issue by eserte** [sisimai/p5-Sisimai#505]
The test suite fails if YAML.pm is not installed (maybe a missing prerequisite specification?):

```
Can't locate YAML.pm in @INC (you may need to install the YAML module) (@INC contains: ./lib ./blib/lib ...  .) at ./t/600-lhost-code line 19.
BEGIN failed--compilation aborted at ./t/600-lhost-code line 19.
Compilation failed in require at t/301-rhost-google.t line 5.
t/301-rhost-google.t ........... 
Dubious, test returned 2 (wstat 512, 0x200)
No subtests run 
... etc ...
```

**Comment by azumakuniyuki** [comment]
@eserte Thanks for the report. I'll try to check and fix the test codes in v5.0.0.

**Comment by azumakuniyuki** [comment]
Tests with the fixed code have been completed successfully at a host without YAML.pm

```
# perl -v

This is perl 5, version 38, subversion 2 (v5.38.2) built for x86_64-linux
```

```
# perldoc -l YAML
No documentation found for "YAML".

# perl -MYAML -lE ''
Can't locate YAML.pm in @INC (you may need to install the YAML module) (@INC entries checked: /usr/local/lib/perl5/site_perl/5.38.2/x86_64-linux /usr/local/lib/perl5/site_perl/5.38.2 /usr/local/lib/perl5/5.38.2/x86_64-linux /usr/local/lib/perl5/5.38.2 /usr/local/lib/perl5/site_perl/5.24.1 /usr/local/lib/perl5/site_perl).
BEGIN failed--compilation aborted.
```

```
# git clone https://github.com/sisimai/p5-sisimai.git
...
# cd ./p5-sisimai
# make test
prove -Ilib --timer t/
[17:19:53] t/000-compile.t ................ ok      357 ms ( 0.02 usr  0.00 sys +  0.26 cusr  0.06 csys =  0.34 CPU)
[17:19:53] t/001-sisimai.t ................ 1/?  ***warning: Sisimai->make will be removed at v5.1.0. Use Sisimai->rise instead at lib/Sisimai.pm line 11.
...
[17:26:33] t/899-arf.t .................... ok      558 ms ( 0.09 usr  0.00 sys +  0.42 cusr  0.04 csys =  0.55 CPU)
[17:26:34]
All tests successful.
Files=104, Tests=96534, 40 wallclock secs ( 4.71 usr  0.24 sys + 30.72 cusr  3.04 csys = 38.71 CPU)
Result: PASS
...
prove -Ilib --timer xt/
[17:26:34] xt/001-used-modules.t ........... skipped: No Test::UsedModules
[17:26:34] xt/002-sample-emails.t .......... ok       69 ms ( 0.00 usr  0.00 sys +  0.06 cusr  0.01 csys =  0.07 CPU)
[17:26:34] xt/610-lhost-activehunter.t ..... skipped: ./set-of-emails/private/lhost-activehunter not found
...
[17:26:43] xt/899-arf.t .................... skipped: ./set-of-emails/private/arf not found
[17:26:43]
All tests successful.
Files=60, Tests=1,  9 wallclock secs ( 0.15 usr  0.07 sys +  7.69 cusr  1.28 csys =  9.19 CPU)
Result: PASS
```

**Comment by azumakuniyuki** [comment]
This issue fixed at v5.0.1
